.github/workflows/release.yml

```diff
@@ -0,0 +1,88 @@
+name: release-muggle
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+      - '*.*.*-*'
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build package using Poetry and store result
+        uses: chaoss/grimoirelab-github-actions/build@master
+        with:
+          artifact-name: sortinghat-dist
+          artifact-path: dist
+
+  tests:
+    needs: [build]
+    runs-on: ubuntu-latest
+    services:
+      mysql:
+        image: mysql:5.7
```

Comment:
Can we use mariadb instead?

Reply:
I changed the version to `mariadb:10.5`, do you prefer any other version of mariadb?

Reply:
That one is fine. Thanks!